protocol/transport/transport.go

```diff
@@ -219,15 +220,19 @@ func (t *Transport) RetrieveRawAll() (map[Filter][]*types.Message, error) {
 
 	allFilters := t.filters.Filters()
 	for _, filter := range allFilters {
-		// Don't pull from filters we don't listen to
-		if !filter.Listen {
-			continue
-		}
 		msgs, err := t.api.GetFilterMessages(filter.FilterID)
 		if err != nil {
 			logger.Warn("failed to fetch messages", zap.Error(err))
 			continue
 		}
+		// Don't pull from filters we don't listen to
```

Comment:
is the comment still relevance? it seems we do pull now. Also should we remove the code line 222?

Reply:
not relevant anymore, you are right

Reply:
and yes, I'll remove the code, thanks!